web/ui/src/lib/clustersMap/campus/wolfsburg.ts

```diff
@@ -0,0 +1,67 @@
+import { Campus } from '../campus';
+import { Cluster } from '../cluster';
+import { CampusNames, ICampus } from '../types';
+
+//
+export class Malaga extends Campus implements ICampus {
```

Comment:
I don't see that, wrong class naming 👀

ℹ️ You can run `yarn make-pretty` to lint your changes